core/src/main/scripts/importer/validateData.py

```diff
@@ -1895,6 +1895,9 @@ def checkLine(self, data):
                                'column_number': col_index + 1,
                                'cause': value})
             elif col_name in ('loc.start', 'loc.end'):
+                # convert possible scientific notation to python scientific notation
+                if "e+" in value:
+                    value = float(value.replace("e+", "e"))
                 try:
                     parsed_coords[col_name] = int(value)
                 except ValueError:
```

Comment:
Another issue here: please log the appropriate message about the validity of the data file format if this field is not a floating-point number instead of letting the validation script crash with an exception, and also log a message if the floating-point number that does not make sense as a genomic coordinate.